introduction.md

```diff
@@ -0,0 +1 @@
+Hi. This is an introduction. I really like things. I really, really do.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.